pillar.example

```diff
@@ -84,6 +85,8 @@ nginx:
       - upstream netdata:
         - server: 127.0.0.1:19999
         - keepalive: 64
+    file_retrieved_by_tofs: {} # File retrieved by TOFS,
```

Comment:
How would this be used exactly?

Reply:
Don't know how to explain that, it will search files :

Reply:
Isn't this already provided by the `tofs` section of the pillar?  I.e.:

```sls
  tofs:
    ...
    source_files:
      ...
      nginx_snippet_file_managed:
        - file_retrieved_by_tofs
        - server.conf
```

Reply:
This is the default behavior without specifying anything in pillar